examples/manifold/plot_mds.py

```diff
@@ -68,7 +69,7 @@
 plt.scatter(npos[:, 0], npos[:, 1], color='darkorange', s=s, lw=0, label='NMDS')
 plt.legend(scatterpoints=1, loc='best', shadow=False)
 
-similarities = similarities.max() / similarities * 100
+similarities = similarities.max() / (similarities + EPSILON) * 100
 similarities[np.isinf(similarities)] = 0
```

Comment:
If we use `EPSILON`, then `similarities[np.isinf(similarities)] = 0` can be removed.

Reply:
Okay, I will remove it. thanks